### PR TITLE
Canes + Cane Blade for Syndicate Librarians

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

Added canes to the CuraDrobe. I also added a cane that can be unsheathed to reveal a blade for syndicate librarians. Un Gurde! Have at thee!
## Why / Balance

More syndicate melee weapons are nice, also something for librarians. Maybe someone in the future can make a old trait where if your character is too-old they need a cane? Or just something to be the stations pimp.

## Media
image
image
Screenshot 2024-03-05 022853

- [X] I have added screenshots/videos to this PR showcasing its changes ingame
**Changelog**

:cl:
- add: Added Canes to the CuraDrobe and Cane Blades for Syndicate Librarians

### CODE REVIEW

**Comment by lzk228** [comment]
shooting cane for mime?

**Comment by ps3moira** [comment]
> shooting cane for mime?

No but I'm thinking on making a shotgun umbrella and umbrellas being a thing.

**Comment by Hmeister-real** [comment]
is there a normal cane for normal librarians?

**Comment by TheShuEd** [comment]
oh my god, yeah, more secular stuff.

**Comment by ps3moira** [comment]
> is there a normal cane for normal librarians?

Normal canes can be found in the CuraDrobe while the Cane Blade can be found in the uplink for librarians.

**Comment by TheShuEd** [comment]
test fail is real, fix it

**Comment by ElPidor** [comment]
Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves.  
In 2 words 
Very weak as fuck weapon
**REALLY need buffs**

**Comment by ps3moira** [comment]
> Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves. In 2 words Very weak as fuck weapon **REALLY need buffs**

This weapon is more meant to be a one-on-one stealth weapon not a kill everyone super easily and replace the dual-sword kinda weapon.

**Comment by SlamBamActionman** [comment]
> > Literraly very garbage item to shit into 20 tc bundle. Up damage to 20 per slash\or give it more reflection, let it apply as hell more bleeding or some special ability _blocking_ etc. I can give my 5 tc to assistant to get my free 6 tc stanbaton AK boxer gloves. In 2 words Very weak as fuck weapon **REALLY need buffs**
> 
> This weapon is more meant to be a one-on-one stealth weapon not a kill everyone super easily and replace the dual-sword kinda weapon.

I think there is a point that the stats are kinda underwhelming right now. Compare the damage to other stealth/improvised weapons in the uplink: 

- Cybersun Pen: 15 damage, very silent, 1 TC
- Syndie Toolbox: 20 damage, bonk, 2 TC
- Surgical bag: Multiple tools like the saw, 15 damage, 4 TC

My suggestion would be to bump it down in TC and bring it in line with the other options damage-wise. Maybe 17 slash, since it is on par with the pen for concealability but can't edit stamped documents? If you want to make it have some combat flair, you can decrease the damage but give it a faster swing speed to keep the DPS high.

All that being said, cool weapon!

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    attackRate: 1.2
+    damage:
+      types:
+        Slash: 12
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
+  - type: Reflect
+    enabled: true
+    reflectProb: .3
+    spread: 90
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:50]: Shouldn't be reflecting.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml (hunk 1 above)
Comment (Hmeister-real) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:50]: shouldn't it be? Its more tc than an e-dagger, and pretty meh even with the reflect.

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:50]: Energy weapons reflecting is intuitive but random bladed weapons doing it is not.

Comment (ps3moira) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml:50]: Fair

**Comment by ps3moira** [comment]
I added changes to it's stats. Removed the reflect and buffed the damage 12->14 and attack rate 1.2->1.5. If there's need for more stat changes, I can do so.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**